src/features/bondfeatures.jl

```diff
@@ -0,0 +1,18 @@
+include("abstractfeatures.jl")
+
+abstract type BondFeature <: AbstractPairFeature end
```

Comment:
I do think this overall structure makes sense for the reasons we talked about – easy to connect things like `BondType`, `IsConjugated`, etc. to the relevant rdkit functions without clunky dictionary stuff